src/terminal/color-scheme.c

```diff
@@ -46,7 +46,7 @@
  * @return
  *     Zero if the two strings are equal and non-zero otherwise.
  */
-static int guac_terminal_color_scheme_compare_token(const char* str_start,
+static int guac_terminal_get_color_scheme_compare_token(const char* str_start,
         const char* str_end, const char* literal) {
```

Comment:
While this does return an `int`, it's not a getter; it's a comparator. I think the old naming was correct.

Reply:
Ah, sorry. Over-agressive find/replace here. Fixed.